spec/acceptance/itk_spec.rb

```diff
@@ -7,8 +7,8 @@
   service_name = 'apache2'
   variant = :prefork
 when 'redhat'
-    variant = (os[:release].to_i >= 7) ? :prefork : :itk_only
-    service_name = 'httpd'
+  variant = :prefork
```

Comment:
Note the second part I wrote: every one is now `:prefork`. So you can simplify the whole file.